client/src/legacy/TinyMCE_sslink-email.js

```diff
@@ -9,19 +9,20 @@ import { loadComponent } from 'lib/Injector';
 
 const commandName = 'sslinkemail';
 
-// Link to email address
-TinyMCEActionRegistrar.addAction(
-  'sslink',
-  {
-    text: i18n._t('Admin.LINKLABEL_EMAIL', 'Link to email address'),
-    onclick: (editorInst) => editorInst.execCommand(commandName),
-    priority: 51,
-  },
-  editorIdentifier,
-).addCommandWithUrlTest(commandName, /^mailto:/);
-
 const plugin = {
   init(editor) {
+    // Add "Link to email address" to link menu for this editor
+    TinyMCEActionRegistrar.addAction(
+      'sslink',
+      {
+        text: i18n._t('Admin.LINKLABEL_EMAIL', 'Link to email address'),
+        onclick: (editorInst) => editorInst.execCommand(commandName),
+        priority: 51,
+      },
+      editor.settings.editorIdentifier,
+    ).addCommandWithUrlTest(commandName, /^mailto:/);
```

Comment:
Moving this into here means we have access to the actual editor instance, so we can find its editor identifier.
Note the change from `editorIdentifier` to `editor.settings.editorIdentifier`.